src/signal-forms/signal-input-error.directive.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  AfterViewInit,
+  Directive,
+  inject, InjectionToken, Injector,
+  ViewContainerRef
+} from '@angular/core';
+import {SIGNAL_INPUT_ERROR_COMPONENT} from "@signal-form/signal-input-error.token";
+import {SignalInputDirective} from "@signal-form/signal-input.directive";
+import {FormField} from "@signal-form/form-field";
+
+
+const SIGNAL_INPUT_ERROR_FIELD= new InjectionToken<FormField>('Signal input error form field');
```

Comment:
Feel free to also add and export these in the `index.ts` file.
You can then import via `@signal-form`.